gen/gen.c

```diff
@@ -2315,8 +2315,7 @@ usage(void)
 	exit(1);
 }
 
-
-static char *
+char *
```

Comment:
I think it's better to move `timestamp()` to `util.c` to avoid cyclic dependency between `gen.c` and other files like `sequencecheck.c`.